templates/php/src/Client.php.twig

```diff
@@ -176,6 +176,13 @@ class Client
         $responseBody   = curl_exec($ch);
         $contentType    = $responseHeaders['content-type'] ?? '';
         $responseStatus = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+
+        $warnings = $responseHeaders['x-{{spec.title | caseLower}}-warning'] ?? '';
+        if ($warnings) {
+            foreach(explode(';', $warnings) as $warning) {
+                trigger_error($warning, E_USER_WARNING);
```

Comment:
I don't think `trigger_error` is right, I think it throws exception from what I understood here https://www.php.net/manual/en/function.trigger-error.php

Reply:
<img width="1028" alt="Screenshot 2024-05-21 at 10 19 41" src="https://github.com/appwrite/sdk-generator/assets/22452787/ba463d89-6f03-4b81-bef4-f556d450997f">
Seems to behave how we need based on my playground testing

Reply:
Done!